linux/base.Dockerfile

```diff
@@ -202,6 +202,30 @@ RUN wget -nv -O puppet-tools.deb https://apt.puppet.com/puppet-tools-release-bus
   && apt-get install puppet-bolt \
   && rm -f puppet-tools.deb
 
+# install Rust
+
+ENV RUSTUP_HOME=/usr/local/rustup \
+  CARGO_HOME=/usr/local/cargo \
+  PATH=/usr/local/cargo/bin:$PATH
+
+RUN RUN set -eux; \
+  apt-get update; \
+  apt-get install -y --no-install-recommends \
+  ca-certificates \
+  gcc \
+  libc6-dev \
+  wget \
+  ; \
+  \ 
+  wget https://static.rust-lang.org/rustup/dist/x86_64-unknown-linux-gnu/rustup-init \
+  chmod +x rustup-init; \
+  ./rustup-init -y --no-modify-path --default-toolchain nightly; \
```

Comment:
What’s the reason for defaulting to nightly?

Reply:
@rylev good point. That seems to be what Docker Hub image has but it makes more sense to use the stable version imo